Fjernet bruken av react-responsive

fredrik-oh opened the pull request:
Jeg og Isa syntes det var unødvendig å trekke inn en ny dependency for å løse et problem som kan løses vha standard javascript.

Commit:
Fjernet bruken av react-responsive for å begrense pakkemengden i prosjektet.

martonSkjevelandNav commented on src/felleskomponenter/oppsummering/oppsummering.tsx:
```diff
@@ -44,7 +44,7 @@ const Oppsummering = (props: OppsummeringProps) => {
   if (!oppsummering || !fagsak?.sakstype) return <div />;
 
-  const isLitenSkjerm = useMediaQuery({ query: "(max-width: 1440px)" });
+  const isLitenSkjerm = useMediaQuery("(max-width: 1440px)");
```
Når vi først lager vår egen hadde det vært fint med en mer robust query. Kan ta inn max-width som number parameter og lage strengen i den nye funksjonen

Commit:
Endret mediaQuery.tsx til å motta et queryobjekt.
Dersom mer komplekse behov oppstår typ utstrakt bruk av breakpoints burde vi nok gå over til en dependency da dette ikke er en effektiv implementasjon (Hver komponent kjører media queries for hvert resize event)

martonSkjevelandNav reviewed (approved):
👏
